<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathlineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="GoTopushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icon.qrc">
          <normaloff>:/ico/icon/goto.ico</normaloff>:/ico/icon/goto.ico</iconset>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DirSelectpushButton">
        <property name="text">
         <string>选择路径</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBox_filetype"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_filter">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_search_all">
        <property name="text">
         <string>搜索所有文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_select_all">
        <property name="text">
         <string>全选</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionunlockFile"/>
   <addaction name="separator"/>
   <addaction name="actionLockFile"/>
   <addaction name="separator"/>
   <addaction name="actionUnlockDir"/>
   <addaction name="separator"/>
   <addaction name="actionLockDir"/>
   <addaction name="separator"/>
   <addaction name="actionUnlockExt"/>
   <addaction name="separator"/>
   <addaction name="actionLockExt"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionunlockFile">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ico/icon/unlock_file.ico</normaloff>:/ico/icon/unlock_file.ico</iconset>
   </property>
   <property name="text">
    <string>unlockFile</string>
   </property>
   <property name="toolTip">
    <string>解锁文件</string>
   </property>
  </action>
  <action name="actionLockFile">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ico/icon/lock_file.ico</normaloff>:/ico/icon/lock_file.ico</iconset>
   </property>
   <property name="text">
    <string>lockFile</string>
   </property>
   <property name="toolTip">
    <string>锁定文件</string>
   </property>
  </action>
  <action name="actionUnlockDir">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ico/icon/dir_unlock.ico</normaloff>:/ico/icon/dir_unlock.ico</iconset>
   </property>
   <property name="text">
    <string>unlockDir</string>
   </property>
   <property name="toolTip">
    <string>解锁文件夹</string>
   </property>
  </action>
  <action name="actionLockDir">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ico/icon/dir_lock.ico</normaloff>:/ico/icon/dir_lock.ico</iconset>
   </property>
   <property name="text">
    <string>LockDir</string>
   </property>
   <property name="toolTip">
    <string>锁定文件夹</string>
   </property>
  </action>
  <action name="actionLockExt">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ico/icon/lock_ext.ico</normaloff>:/ico/icon/lock_ext.ico</iconset>
   </property>
   <property name="text">
    <string>LockExt</string>
   </property>
   <property name="toolTip">
    <string>锁定指定的文件后缀</string>
   </property>
  </action>
  <action name="actionUnlockExt">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ico/icon/unlock_ext.ico</normaloff>:/ico/icon/unlock_ext.ico</iconset>
   </property>
   <property name="text">
    <string>UnlockExt</string>
   </property>
   <property name="toolTip">
    <string>解锁指定的文件后缀</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
